packages/agoric-cli/src/follow.js

```diff
@@ -2,16 +2,91 @@
 import process from 'process';
 import { Far, getInterfaceOf } from '@endo/marshal';
 import { decodeToJustin } from '@endo/marshal/src/marshal-justin.js';
+import { assert, details as X } from '@agoric/assert';
 
 import {
+  DEFAULT_KEEP_POLLING_SECONDS,
+  DEFAULT_JITTER_SECONDS,
   iterateLatest,
   makeCastingSpec,
   iterateEach,
   makeFollower,
   makeLeader,
 } from '@agoric/casting';
+import { Command } from 'commander';
 import { makeLeaderOptions } from './lib/casting.js';
 
+export const createFollowCommand = () =>
```

Comment:
+1 to using `Command`. though with it, this file should go in `src/commands`. You might want do the Commander refactorings in a separate PR.

Reply:
hm... using `src/commands` seems to be the exception, not the rule. `main.js` looks like:

```js
import cosmosMain from './cosmos.js';
import deployMain from './deploy.js';
import publishMain from './main-publish.js';
import initMain, { makeInitCommand } from './init.js';
import installMain from './install.js';
import setDefaultsMain, { makeSetDefaultsCommand } from './set-defaults.js';
import startMain from './start.js';
import followMain, { createFollowCommand } from './follow.js';
import { makeOpenCommand, makeTUI } from './open.js';
import { makeWalletCommand } from './commands/wallet.js';
```

Reply:
Depends on how you look at it. What I am requesting is that files that import the `Command` class (and export an instance of it) go in `src/commands`. There are 4 examples of that in master. The one exception in `src` is `src/main.js` which is making the program. That's cool.

These others are not command instances:
```
import cosmosMain from './cosmos.js';
import deployMain from './deploy.js';
import publishMain from './main-publish.js';
import installMain from './install.js';
import startMain from './start.js';
```

And these in your snippet actually are Commands, but ones you're making in this PR:
```
// makeInitCommand
// makeSetDefaultsCommand
// createFollowCommand
// makeOpenCommand
```
So I'm requesting that they go with the other Command objects in `src/commands`.

Reply:
I don't plan to export an instance of `Command`; I plan to export a maker function that takes the necessary authority as args and creates a `Command`.

Reply:
Sorry for my impression: exporting a maker function that returns an instance of `Command`. That's what `createFollowCommand` is, right? That's also what these are:

https://github.com/Agoric/agoric-sdk/blob/f1ff7b99bfea6841e7627987bd482b8242caf207/packages/agoric-cli/src/commands/psm.js#L59
https://github.com/Agoric/agoric-sdk/blob/f1ff7b99bfea6841e7627987bd482b8242caf207/packages/agoric-cli/src/commands/perf.js#L26
https://github.com/Agoric/agoric-sdk/blob/f1ff7b99bfea6841e7627987bd482b8242caf207/packages/agoric-cli/src/commands/oracle.js#L20
https://github.com/Agoric/agoric-sdk/blob/f1ff7b99bfea6841e7627987bd482b8242caf207/packages/agoric-cli/src/commands/wallet.js#L31

When you wrote "seems to be the exception, not the rule" I took it that you were appealing to precedent. The for functions like you describe (returning a Command instance) is that all four of them are in `src/commands`. It seems important to you that you don't put those files in `src/commands`. I don't understand why but I'll wait for R4R before engaging on the topic again in this PR.